src/blocks/modals/ContentHashModal.vue

```diff
@@ -4,8 +4,8 @@
     <p>
       A content hash is a cryptographic hash of raw content. Different systems have different ways of turning this hash (called a digest) into an identifier that their system uses
       to locate content. The most popular decentralized data storage services are IPFS and Arweave.
-      <br /><br />No experience with content hashes? Use <a href="https://zknft.xyz/" target="_blank">this tool</a>! It will automagically upload your content to IPFS and mint it
-      to your zkSync account.
+      <br /><br />No experience with content hashes? Use <a href="https://ipfs.io/#install" target="_blank">IPFS tools</a>! After uploading your content and getting the content
+      hash, zkSync Wallet will automagically mint an NFT to your zkSync account.
```

Comment:
`After uploading your content and getting the content hash, zkSync Wallet will automagically mint an NFT to your zkSync account.`. I think that the wording should be changed.

1. It will not automatically mint anything. Users still have to enter the content hash here.
2. In my experience, IPFS hash is usually represented by a base64-encoded string. So users will have to decode it to make sure they can mint an NFT.